.buildkite/pipeline.yml

```diff
@@ -203,3 +203,16 @@ steps:
           image: "rustvmm/dev:v6"
           always-pull: true
           propagate-environment: true
+
+  - label: "cargo-audit"
+    commands:
+      - pytest rust-vmm-ci/integration_tests/test_cargo_audit.py -s
```

Comment:
On a second thought, we can just write `cargo audit` here and it should work :-?
This PR was a :roller_coaster: